actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -622,19 +622,19 @@ def method_missing(method, *args, &block)
   # the request format to JSON unless overridden), sets the content type to
   # "application/json" and encodes the parameters as JSON.
   #
-  # Calling TestResponse#parsed_body on the response parses the response body based on the
+  # Calling ActionDispatch::TestResponse#parsed_body on the response parses the response body based on the
```

Comment:
@zzak i made this mention more explicit and fully qualified in the hopes of Rdoc auto-linking to the class page, but I'm not having much luck. Am I doing it wrong?

Reply:
```suggestion
  # Calling TestResponse#parsed_body on the response parses the response body based on the
```

You don't need to prepend the `ActionDispatch` because it's within the same namespace, RDoc will know how to find it.

Tested locally on your branch:
<img width="990" alt="Screenshot 2023-10-07 at 9 49 41" src="https://github.com/rails/rails/assets/277819/b93976e3-95d7-4943-8817-a36aad576327">


<img width="719" alt="Screenshot 2023-10-07 at 9 49 54" src="https://github.com/rails/rails/assets/277819/810fab56-9dc1-4f18-b7a4-e8fb9833a40c">